examples/dmodex.c

```diff
@@ -227,16 +228,16 @@ int main(int argc, char **argv)
         }
     }
 
+done:
+    /* finalize us */
+
     /* call fence so everyone waits before leaving */
-    proc.rank = PMIX_RANK_WILDCARD;
-    if (PMIX_SUCCESS != (rc = PMIx_Fence(&proc, 1, NULL, 0))) {
+    if (PMIX_SUCCESS != (rc = PMIx_Fence(NULL, 0, NULL, 0))) {
```

Comment:
Just an FYI (so no need to change anything): `PMIx_Fence` with a `NULL` proc argument defaults to "all procs in my nspace".